app/collections/create-form.jsx

```diff
@@ -33,13 +37,15 @@ class CollectionsCreateForm extends React.Component {
 
     const collection = {
       display_name: displayName,
+      description,
       private: isPrivate,
       links
     };
 
     apiClient.type('collections').create(collection).save()
       .then((newCollection) => {
         this.name.value = '';
+        this.description.value = '';
         this.isPrivate.value = true;
```

Comment:
As I mentioned in the conversion PR, I think this is in error. This is invalid HTML. If we're resetting the form state after submission, then this should be resetting what the checked attribute is set to.